charts/nfs-subdir-external-provisioner/README.md

```diff
@@ -69,7 +69,7 @@ The following tables lists the configurable parameters of this chart and their d
 | `leaderElection.enabled`            | Enables or disables leader election                                                                   | `true`                                                   |
 | `nfs.server`                        | Hostname of the NFS server (required)                                                                 | null (ip or hostname)                                    |
 | `nfs.path`                          | Basepath of the mount point to be used                                                                | `/nfs-storage`                                           |
-| `nfs.mountOptions`                  | Mount options (e.g. 'nfsvers=3')                                                                      | null                                                     |
+| `nfs.mountOptions`                  | Mount options (e.g. {"nolock"\,"nfsvers=3"})                                                          | null                                                     |
```

Comment:
the `\` disappears when file is rendered as you can see here: https://github.com/kubernetes-sigs/nfs-subdir-external-provisioner/blob/27d772798deb510c105b05618686f35cf0cb71ec/charts/nfs-subdir-external-provisioner/README.md

you can just remove it i think

Reply:
good call. needed double slash to escape.

Reply:
i am wondering, the slash required only in helm `--set` because helm reads the comma as a separator between variables and the slash escapes it. but in this context maybe `{"nolock","nfsvers=3"}` without escape character is more fit... what do you think?

Reply:
the install example list example with set.  ; Can this be configured somehow else? 
The values possible I would get from the Parameter list so if the set requires a escape the example should have it imho.